core/src/main/java/io/aiven/klaw/service/ExportImportDataService.java

```diff
@@ -0,0 +1,157 @@
+package io.aiven.klaw.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.aiven.klaw.config.ManageDatabase;
+import io.aiven.klaw.dao.ProductDetails;
+import io.aiven.klaw.dao.UserInfo;
+import io.aiven.klaw.dao.metadata.KwAdminConfig;
+import io.aiven.klaw.dao.metadata.KwData;
+import io.aiven.klaw.dao.metadata.KwRequests;
+import io.aiven.klaw.helpers.HandleDbRequests;
+import java.io.File;
+import java.io.IOException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+import org.jasypt.util.text.BasicTextEncryptor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Service;
+
+/*
+Export Klaw metadata (Admin config, Core data, Requests data) to json files
+ */
+@Slf4j
+@Service
+public class ExportImportDataService {
+
+  public static final String PWD_ALL_USERS = "WelcomeToKlaw!!";
+
+  @Value("${klaw.jasypt.encryptor.secretkey}")
+  private String encryptorSecretKey;
+
+  private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+  private static final String FILE_EXT = ".json";
+  private static final String FILE_PREFIX = "kwmetadata";
+  private static final String ADMIN_CONFIG_PREFIX = "admin_config";
+  private static final String KW_DATA_PREFIX = "kwdata";
+  private static final String KW_REQUEST_DATA_PREFIX = "kwrequests_data";
+  @Autowired ManageDatabase manageDatabase;
+
+  @Value("${klaw.export.file.path:/target}")
+  private String klawExportFilePath;
+
+  @Value("${klaw.export.scheduler.enable:false}")
+  private boolean exportMetadata;
+
+  @Async("metadataExportTaskExecutor")
+  @Scheduled(cron = "${klaw.export.cron.expression:0 0 0 * * ?}")
+  void exportKwMetadataScheduler() {
+    if (!exportMetadata) {
+      return;
+    }
+    exportKwMetadata();
+  }
```

Comment:
A future enhancement might be to do a delta and export out what's new since the last export